### PR TITLE
Added Github Issue Templates

### PR DESCRIPTION
**Pull request by Sneethe**
Following @mcepl [concerns on noise and bloat in the issue tracker,](https://github.com/martanne/vis/issues/1145#issuecomment-1756281015). I added a page to redirect feature request issues to the mailing list. 
In addition I also added a bug report page that prompts the user for useful information. I also thought it would be a good idea to add the mailing list and IRC to the landing page for more potential eyes on.

Currently you can see what these Issue templates look like [here](https://github.com/Sneethe/vis/issues/new/choose).

Please let me know if these are lacking in anyway. Maybe the [Bug Report page](https://github.com/Sneethe/vis/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml) could have some parts added/removed.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: IRC Channel
+    url: https://web.libera.chat/#vis-editor
```
Comment (mcepl) [inline .github/ISSUE_TEMPLATE/config.yml:4]: IRC URL should be `ircs://irc.libera.chat:6697/vis-editor`, shouldn’t it?

**Comment by mcepl** [comment]
Aside from that nit, I think it should be workable.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (rnpnr) [reply .github/ISSUE_TEMPLATE/config.yml:4]: I'm not sure its that important. For the people who actively use IRC I think "`#vis-editor` on Libera" is sufficient without the link.

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: Feature Request
+description: Thank you for your interest in improving vis.
+title: "[Feature Request]:"
+labels: [meta:feature-request]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### The current practise is for feature discussion to take place in the vis [mailing list](https://lists.sr.ht/~martanne/devel).
```
Comment (rnpnr) [inline .github/ISSUE_TEMPLATE/feature_request.yml:9]: I might just reword this to:
> The current practice and preferred location for large feature requests and general design discussion is the vis [mailing list](https://lists.sr.ht/~martanne/devel).

**Review by rnpnr — APPROVED** [review]
Overall I think this good, I will merge it soon if there are no further objections to the comments I left. Thanks for your help!

**Comment by rnpnr** [comment]
Applied in f41860b. Thanks!

**Comment by goyalyashpal** [comment]
hi @ mcepl ninewise rnpnr 

i saw here too: https://github.com/martanne/vis/pull/1018#issuecomment-1331324729

that the commits are manually applied, rather than merging of the pr. 
is there any special reason to it? what is the procedure you are following to accomplish that? 

i am curious as i am always on the look out of how different people, or different repos manage their contributions and collaboration.

**Comment by mcepl** [comment]
On Tue Dec 26, 2023 at 1:42 PM CET, Charles Tamzarian wrote:
> Following @mcepl [concerns on noise and bloat in the issue tracker,](https://github.com/martanne/vis/issues/1145#issuecomment-1756281015). I added a page to redirect feature request issues to the mailing list. 
Thank you for doing this, it is certainly better to have
it than not to, you have to understand the lack of passion
here, because most people working on vis are advocates of
sites like https://sr.ht and email-based development (see
https://lists.sr.ht/~martanne/devel). Martanne opened GitHub in
the beginning, and we have no hundreds of tickets here, which we
cannot get rid of fast enough, but I would be more than happy, if
whole development moved to the email list.

Best,

Matěj

-- 
http://matej.ceplovi.cz/blog/, @***@***.***
GPG Finger: 3C76 A027 CA45 AD70 98B5  BC1D 7920 5802 880B C9D8
 
Perfection is finally attained not when there is no longer
anything to add, but when there is no longer anything to take
away.
  -- Antoine de Saint-Exupery

**Comment by ninewise** [comment]
I usually apply patches from github similar to what is shown in the video linked, so yanking the PR URL, than `curl -LOJ url.patch` and `git am 1160.patch`. Someone recently told me github has allowed to do 'rebase and merge' since some time but it used to be just merging which is where I got the habbit. For vis, since we push to sourcehut (via a local copy), it's just easier to do everything there.
